assets/src/styles/editorOverrides.scss

```diff
@@ -74,7 +74,7 @@
   text-align: center;
   text-decoration: none;
   text-transform: uppercase;
-  color: $white;
+  color: var(--btn-color, $white);
```

Comment:
Shouldn't this also be introduced for the frontend in [the styleguide PR](https://github.com/greenpeace/planet4-styleguide/pull/59/files)?

Reply:
Good one @Inwerpsel ! I'll fix this now